roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -55,6 +55,12 @@
   when:
     - kata_containers_enabled
 
+- name: Build a list of crio runtimes with gVisor runtime
+  set_fact:
+    crio_runtimes: "{{ crio_runtimes + [gvisor_runtime] }}"
+  when:
+    - gvisor_enabled
```

Comment:
I left this part here because I'm actually hoping gVisor will eventually support cri-o when https://github.com/google/gvisor/issues/193 is fixed.

Is there any intention to properly support cri-o in gVisor? If not then I will remove this part from the PR.

Reply:
I guess I'd remove it for now. cri-o will need some work in order to better support runtimes that aren't vanilla containers. It's not just a matter of supporting systemd-cgroups.

Reply:
Understood, in this case I dropped the support for cri-o until gVisor project decides to support it.